Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -2320,3 +2651,648 @@ carouselView state push =
         ]
     , PrimaryButton.view (push <<< UpdateProfileButtonAC) (updateProfileConfig state)
     , PrimaryButton.view (push <<< SkipAccessibilityUpdateAC) (maybeLaterButtonConfig state)]
+----------------------------------------- UPDATED HOME SCREEN -------------------------------------------
+
+homeScreenView :: forall w. (Action -> Effect Unit) -> HomeScreenState -> PrestoDOM (Effect Unit) w
+homeScreenView push state = 
+  relativeLayout
+    [ height $ V (screenHeight unit)
+    , width $ V (screenWidth unit)
+        ][ 
+      linearLayout
+        [ height $ V ((screenHeight unit)/ 3)
+        , width MATCH_PARENT
+        , background Color.blackBlue
+        , padding $ (PaddingTop (safeMarginTop))
+        ][] 
+        , homescreenHeader push state
+        , linearLayout 
+        [ width MATCH_PARENT
+        , height MATCH_PARENT
+        , margin $ MarginTop 20
+        , orientation VERTICAL
+        ][ 
+           coordinatorLayout
+            [ height MATCH_PARENT
+            , width MATCH_PARENT
+            ][ bottomSheetLayout
+                [ height MATCH_PARENT
+                , width MATCH_PARENT
+                , peakHeight $ spy "getPeekHeight " $ if state.data.peekHeight == 0 || state.data.peekHeight == 700 then getPeekHeight state else state.data.peekHeight
```

Comment:
remove spy

Reply:
@aryan-13 can we make the values used here `0` and `700` configurable.

Reply:
@MercyQueen It is not required here, it is already fetching the Peekheight according to the screenHeight